portals/new_admin/portals/client/src/constants/token.ts

```diff
@@ -0,0 +1 @@
+export const AUTHORIZATION_CODE = "code";
```

Comment:
Shall we add an EOF new line here as well?

Reply:
This issue is fixed in the latest file.